File: src/main/java/me/geekTicket/Command/CommandHelp.java
package me.geekTicket.Command;

import me.geekTicket.Utils.Bukkit.ConfigManager;
import me.geekTicket.Utils.Bukkit.LangLoad;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

public class CommandHelp {

    public static void execute(CommandSender sender, Command command, String label, String[] args)
    {
        if (args.length == 0 || args[0].equals("help"))
        {
            sender.sendMessage("§lGeekTicket §8- §7§l帮助导航 §8- §f§lBate");
            for(String list_to_string : ConfigManager.HELP)
            {
                sender.sendMessage(list_to_string);
            }
        }
    }
}

File: src/main/kotlin/me/geekTicket/Command/CommandHelp.kt
package me.geekTicket.Command

import me.geekTicket.Bukkit.LangMain
import org.bukkit.command.CommandSender
import taboolib.common.platform.command.subCommand

object CommandHelp: CommandExp {
    override val command = subCommand {

        execute<CommandSender> { sender, context, argument ->
            sender.sendMessage("§lGeekTicket §8- §7§l帮助导航 §8- §f§lBate")
            for (list_to_string in LangMain.HELP) {
                sender.sendMessage(list_to_string)
            }
        }
    }
}